manifests/charts/base/files/gen-istio-cluster.yaml

```diff
@@ -4665,14 +4654,6 @@ rules:
     resources: ["tokenreviews"]
     verbs: ["create"]
 
-  # TODO: remove, no longer needed at cluster
-  - apiGroups: [""]
-    resources: ["secrets"]
-    verbs: ["create", "get", "watch", "list", "update", "delete"]
```

Comment:
why not? doesn't istiod create secret?

Reply:
Yeah but only in istio-system - that is why its moved to the Role. The only cluster wide secret reading we need is multicluster secrets - maybe.. see https://github.com/istio/istio/pull/25121#issuecomment-651943274

Reply:
Why would we need it for MC ? They should be in istio-system as well.

Reply:
I agree which is why I have this pr - but in theory today you can have it anywhere

Reply:
if you set a flag